src/test/java/seedu/address/commons/util/AppUtilTest.java

```diff
@@ -9,7 +9,7 @@ public class AppUtilTest {
 
     @Test
     public void getImage_exitingImage() {
-        assertNotNull(AppUtil.getImage("/images/address_book_32.png"));
+        assertNotNull(AppUtil.getImage("/images/clipboard.png"));
```

Comment:
```suggestion
        assertNotNull(AppUtil.getImage("/images/clipboard.PNG"));
```
Seems like this line is causing error